src/aktualizr_secondary/aktualizr_secondary_config.h

```diff
@@ -5,12 +5,7 @@
 #include <boost/program_options.hpp>
 #include <boost/property_tree/ini_parser.hpp>
 
-#include "bootloader/bootloader_config.h"
-#include "crypto/keymanager_config.h"
-#include "crypto/p11_config.h"
-#include "logging/logging_config.h"
-#include "package_manager/packagemanagerconfig.h"
-#include "storage/storage_config.h"
+#include "libaktualizr/config.h"
```

Comment:
It is slightly unfortunate that we have to include a bunch more stuff in aktualizr-secondary that we won't actually use, but for now that seems like a small issue that we can address at another time.